docs/platform-feature-parity.md

```diff
@@ -0,0 +1,173 @@
+| body                | x       | wp-pt   | wp-pt     |
+| username            | x       | w-nt    | wp-pt     |
+| password            | x       | w-nt    | wp-pt     |
+|                     |         |         |           |
+| **interface**       | x       | ni      | ni        |
+| exists              | x       | ni      | ni        |
+| addrs               | x       | ni      | ni        |
+| mtu                 | x       | ni      | ni        |
+|                     | x       |         |           |
+| **kernel-param**    | x       | n/a     | n/a       |
+| value               | x       | n/a     | n/a       |
+|                     | x       |         |           |
+| **mount**           | x       | ni      | ni        |
+| exists              | x       | ni      | ni        |
+| opts                | x       | ni      | n/a       |
+| source              | x       | ni      | n/a       |
+| filesystem          | x       | ni      | ni        |
+| usage               | x       | ni      | ni        |
+|                     | x       |         |           |
+| **matching**        | x       |         |           |
+|                     | x       |         |           |
+| **package**         | x       | ni      | ni        |
+| installed           | x       | ni      | ni        |
+| versions            | x       | ni      | ni        |
+|                     | x       |         |           |
+| **port**            | x       | ni      | ni        |
+| listening           | x       | ni      | ni        |
+| ip                  | x       |         |           |
+|                     | x       |         |           |
+| **process**         | x       | wp-pt   | wp-pt     |
+| running             | x       | wp-pt   | wp-pt     |
+|                     | x       |         |           |
+| **service**         | x       | ni      | ni        |
+| enabled             | x       | ni      | ni        |
+| running             | x       | ni      | ni        |
+|                     | x       |         |           |
+| **user**            | x       | ni      | ni        |
+| exists              | x       | ni      | ni        |
+| uid                 | x       | ni      | n/a       |
+| gid                 | x       | ni      | n/a       |
+| groups              | x       | ni      | ni        |
+| home                | x       | ni      | ni        |
+| shell               | x       | ni      | ni        |
+
+## Matrix - `command`s
+
+| Test       | `linux` | `macOS` | `Windows` |
+|:-----------|---------|---------|-----------|
+| `add`      | x       |         | wp-pt     |
+| `autoadd`  | x       |         |           |
+| `help`     | x       |         | wp-pt     |
+| `render`   | x       |         |           |
+| `serve`    | x       | w-nt    |           |
+| `validate` | x       | w-nt    | wp-pt     |
+
+### `command` testing notes
+
+### Command: `add`
+
+#### Windows `add`
+
+```powershell
```

Comment:
curious.. why does travis-CI not show this message?

Reply:
The goss `log.` output goes to `stderr` and (apparently, I didn't know this until just now) travis doesn't capture `stderr`.

You can try this out locally:

```bash
TRAVIS_OS_NAME="windows "ci/build.sh "windows "> stdout.txt
```

and you will see only the `stderr` output since `stdout` will now appear only within `stdout.txt`.

Here is the output I get:

```
Pete@monsterbox ~/src/improbable/goss (multi-os-binaries)
λ TRAVIS_OS_NAME=windows ci/build.sh windows > stdout.txt
/usr/bin/sh: golint: command not found
2020/06/10 10:16:42 WARNING: goss for this platform ("Windows") is alpha-quality, work-in-progress, and not yet exercised within continuous integration.

You should not expect everything to work. Treat linux as the canonical behaviour to expect.

Please see https://github.com/aelsabbahy/goss/tree/master/docs/platform-feature-parity.md to set your expectations and see progress.
Please file issues via https://github.com/aelsabbahy/goss/issues/new/choose
Pull requests and bug reports very welcome.
2020/06/10 10:16:42 WARNING: goss for this platform ("Windows") is alpha-quality, work-in-progress, and not yet exercised within continuous integration.

You should not expect everything to work. Treat linux as the canonical behaviour to expect.

Please see https://github.com/aelsabbahy/goss/tree/master/docs/platform-feature-parity.md to set your expectations and see progress.
Please file issues via https://github.com/aelsabbahy/goss/issues/new/choose
Pull requests and bug reports very welcome.
2020/06/10 10:16:42 WARNING: goss for this platform ("Windows") is alpha-quality, work-in-progress, and not yet exercised within continuous integration.

You should not expect everything to work. Treat linux as the canonical behaviour to expect.

Please see https://github.com/aelsabbahy/goss/tree/master/docs/platform-feature-parity.md to set your expectations and see progress.
Please file issues via https://github.com/aelsabbahy/goss/issues/new/choose
Pull requests and bug reports very welcome.
2020/06/10 10:16:42 WARNING: goss for this platform ("Windows") is alpha-quality, work-in-progress, and not yet exercised within continuous integration.

You should not expect everything to work. Treat linux as the canonical behaviour to expect.

Please see https://github.com/aelsabbahy/goss/tree/master/docs/platform-feature-parity.md to set your expectations and see progress.
Please file issues via https://github.com/aelsabbahy/goss/issues/new/choose
Pull requests and bug reports very welcome.
2020/06/10 10:16:42 WARNING: goss for this platform ("Windows") is alpha-quality, work-in-progress, and not yet exercised within continuous integration.

You should not expect everything to work. Treat linux as the canonical behaviour to expect.

Please see https://github.com/aelsabbahy/goss/tree/master/docs/platform-feature-parity.md to set your expectations and see progress.
Please file issues via https://github.com/aelsabbahy/goss/issues/new/choose
Pull requests and bug reports very welcome.
2020/06/10 10:16:42 WARNING: goss for this platform ("Windows") is alpha-quality, work-in-progress, and not yet exercised within continuous integration.

You should not expect everything to work. Treat linux as the canonical behaviour to expect.

Please see https://github.com/aelsabbahy/goss/tree/master/docs/platform-feature-parity.md to set your expectations and see progress.
Please file issues via https://github.com/aelsabbahy/goss/issues/new/choose
Pull requests and bug reports very welcome.
2020/06/10 10:16:43 WARNING: goss for this platform ("Windows") is alpha-quality, work-in-progress, and not yet exercised within continuous integration.

You should not expect everything to work. Treat linux as the canonical behaviour to expect.

Please see https://github.com/aelsabbahy/goss/tree/master/docs/platform-feature-parity.md to set your expectations and see progress.
Please file issues via https://github.com/aelsabbahy/goss/issues/new/choose
Pull requests and bug reports very welcome.
2020/06/10 10:16:43 WARNING: goss for this platform ("Windows") is alpha-quality, work-in-progress, and not yet exercised within continuous integration.

You should not expect everything to work. Treat linux as the canonical behaviour to expect.

Please see https://github.com/aelsabbahy/goss/tree/master/docs/platform-feature-parity.md to set your expectations and see progress.
Please file issues via https://github.com/aelsabbahy/goss/issues/new/choose
Pull requests and bug reports very welcome.
2020/06/10 10:16:43 WARNING: goss for this platform ("Windows") is alpha-quality, work-in-progress, and not yet exercised within continuous integration.

You should not expect everything to work. Treat linux as the canonical behaviour to expect.

Please see https://github.com/aelsabbahy/goss/tree/master/docs/platform-feature-parity.md to set your expectations and see progress.
Please file issues via https://github.com/aelsabbahy/goss/issues/new/choose
Pull requests and bug reports very welcome.
2020/06/10 10:16:43 WARNING: goss for this platform ("Windows") is alpha-quality, work-in-progress, and not yet exercised within continuous integration.

You should not expect everything to work. Treat linux as the canonical behaviour to expect.

Please see https://github.com/aelsabbahy/goss/tree/master/docs/platform-feature-parity.md to set your expectations and see progress.
Please file issues via https://github.com/aelsabbahy/goss/issues/new/choose
Pull requests and bug reports very welcome.
2020/06/10 10:16:43 WARNING: goss for this platform ("Windows") is alpha-quality, work-in-progress, and not yet exercised within continuous integration.

You should not expect everything to work. Treat linux as the canonical behaviour to expect.

Please see https://github.com/aelsabbahy/goss/tree/master/docs/platform-feature-parity.md to set your expectations and see progress.
Please file issues via https://github.com/aelsabbahy/goss/issues/new/choose
Pull requests and bug reports very welcome.
2020/06/10 10:16:43 WARNING: goss for this platform ("Windows") is alpha-quality, work-in-progress, and not yet exercised within continuous integration.

You should not expect everything to work. Treat linux as the canonical behaviour to expect.

Please see https://github.com/aelsabbahy/goss/tree/master/docs/platform-feature-parity.md to set your expectations and see progress.
Please file issues via https://github.com/aelsabbahy/goss/issues/new/choose
Pull requests and bug reports very welcome.
2020/06/10 10:16:43 WARNING: goss for this platform ("Windows") is alpha-quality, work-in-progress, and not yet exercised within continuous integration.

You should not expect everything to work. Treat linux as the canonical behaviour to expect.

Please see https://github.com/aelsabbahy/goss/tree/master/docs/platform-feature-parity.md to set your expectations and see progress.
Please file issues via https://github.com/aelsabbahy/goss/issues/new/choose
Pull requests and bug reports very welcome.
2020/06/10 10:16:43 WARNING: goss for this platform ("Windows") is alpha-quality, work-in-progress, and not yet exercised within continuous integration.

You should not expect everything to work. Treat linux as the canonical behaviour to expect.

Please see https://github.com/aelsabbahy/goss/tree/master/docs/platform-feature-parity.md to set your expectations and see progress.
Please file issues via https://github.com/aelsabbahy/goss/issues/new/choose
Pull requests and bug reports very welcome.
2020/06/10 10:16:43 WARNING: goss for this platform ("Windows") is alpha-quality, work-in-progress, and not yet exercised within continuous integration.

You should not expect everything to work. Treat linux as the canonical behaviour to expect.

Please see https://github.com/aelsabbahy/goss/tree/master/docs/platform-feature-parity.md to set your expectations and see progress.
Please file issues via https://github.com/aelsabbahy/goss/issues/new/choose
Pull requests and bug reports very welcome.
2020/06/10 10:16:43 WARNING: goss for this platform ("Windows") is alpha-quality, work-in-progress, and not yet exercised within continuous integration.

You should not expect everything to work. Treat linux as the canonical behaviour to expect.

Please see https://github.com/aelsabbahy/goss/tree/master/docs/platform-feature-parity.md to set your expectations and see progress.
Please file issues via https://github.com/aelsabbahy/goss/issues/new/choose
Pull requests and bug reports very welcome.
2020/06/10 10:16:43 WARNING: goss for this platform ("Windows") is alpha-quality, work-in-progress, and not yet exercised within continuous integration.

You should not expect everything to work. Treat linux as the canonical behaviour to expect.

Please see https://github.com/aelsabbahy/goss/tree/master/docs/platform-feature-parity.md to set your expectations and see progress.
Please file issues via https://github.com/aelsabbahy/goss/issues/new/choose
Pull requests and bug reports very welcome.
2020/06/10 10:16:43 WARNING: goss for this platform ("Windows") is alpha-quality, work-in-progress, and not yet exercised within continuous integration.

You should not expect everything to work. Treat linux as the canonical behaviour to expect.

Please see https://github.com/aelsabbahy/goss/tree/master/docs/platform-feature-parity.md to set your expectations and see progress.
Please file issues via https://github.com/aelsabbahy/goss/issues/new/choose
Pull requests and bug reports very welcome.
```